app/views/new_contact_message_mailer/contact_message_notification.html.erb

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <h1 style="color:#363691">A visitor to Any Rat Rescue has sent you a message.</h1>
+    <div>
+      <table style="border-collapse: collapse;">
+        <% ContactMessage::FORM_FIELDS.each do |field| %>
```

Comment:
I have an idea of what you wanted to do with that helper file now? Thoughts on moving the constant access over to the helper?